.ci/magician/github/get.go

```diff
@@ -116,7 +116,7 @@ func (gh *Client) GetCommitMessage(owner, repo, sha string) (string, error) {
 		} `json:"commit"`
 	}
 
-	err := utils.RequestCall(url, "GET", gh.token, &commit, nil)
+	err := utils.RequestCallSilent(url, "GET", gh.token, &commit, nil)
```

Comment:
Why does this need to be silent? We generally want to log all of these requests for debugging purposes.

Reply:
Thomas remove the printing of the result for POST requests because this API outputs a lot. I am pretty sure we are listing a lot of commits so I can understand.

I added the printing of the POST back but thomas was saying that this caused the test output to be too long. I don't think we should be testing this in a presubmit anyways but this is just to unblock adding back the POST result to console